### PR TITLE
infisical: set official infisical team as maintainers

### PR DESCRIPTION
**Pull request by jgoux**
## Description of changes

@akhilmhdh from the Infisical team accepted to maintain the package officially.

I'm removing myself and the other maintainer.

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by akhilmhdh** [comment]
Yes please. We are happy to take it over.

Thank you @jgoux for maintaining this till now.

File: maintainers/team-list.nix [hunk 1]
```diff
@@ -467,6 +467,14 @@ with lib.maintainers; {
     shortName = "Home Assistant";
   };
 
+  infisical = {
+    members = [
+      akhilmhdh
+    ];
+    scope = "Maintain Infisical.";
```
Comment (Aleksanaa) [inline maintainers/team-list.nix:474]:
```suggestion
    scope = "Maintain Infisical";
```

**Comment by Aleksanaa** [comment]
Have you asked @ivanmoreau?

**Comment by jgoux** [comment]
> Have you asked @ivanmoreau?

I didn't, Ivan wrote the initial script and I reworked it to automate the updates and accepted all the PRs since then.

We can wait for his feedback. 👍

**Comment by Aleksanaa** [comment]
@akhilmhdh Just a small reminder: the maintainer list is usually not exclusive, i.e. if someone has made meaningful contributions to a package, we cannot prevent them from adding their name to the maintainer list.

**Comment by akhilmhdh** [comment]
Hey @Aleksanaa We don't mind that but if possible some kinda of comment regarding whose the main stakeholder / primary contact point something of those sort would be helpful. 

Like we don't want our customers to ping a person who has no direct relation. Not a biggie but still a concern, that's all

**Pull request merged**